.github/CODEOWNERS

```diff
@@ -1,12 +1,20 @@
 # Automatically request reviews when a pull request changes any owned files
 # More information: https://github.com/blog/2392-introducing-code-owners 
 
-*.groovy @dotnet/roslyn-infrastructure
+*.yml @dotnet/roslyn-infrastructure
 .github/ @dotnet/roslyn-infrastructure
-build/ @dotnet/roslyn-infrastructure
+eng/ @dotnet/roslyn-infrastructure
```

Comment:
care to mention what eng stands for? looks like eng and scripts directories have a conceptual overlap?

Reply:
I have no idea. @tmat might know.